src/middle/Stan_math_signatures.ml

```diff
@@ -1102,6 +1110,14 @@ let () =
     add_unqualified
       ("num_elements", ReturnType UInt, [bare_array_type (UVector, i)])
   done ;
+  add_unqualified
+    ( "ordered_logistic_glm_lpmf"
+    , ReturnType UReal
+    , [bare_array_type (UInt, 1); UMatrix; UVector; UVector] ) ;
```

Comment:
Same thing here. These are the support signatures: 

real ordered_logistic_glm_lpmf(int, matrix, vector, vector)
real ordered_logistic_glm_lpmf(int, row_vector, vector, vector)
real ordered_logistic_glm_lpmf(int[], matrix, vector, vector)
real ordered_logistic_glm_lpmf(int[], row_vector, vector, vector)